VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/data/Credentials.java

```diff
@@ -1,11 +1,26 @@
 package org.edx.mobile.test.feature.data;
 
+import android.support.annotation.Nullable;
+import org.edx.mobile.test.feature.config.E2EConfig;
+import org.edx.mobile.util.ResourceUtil;
+
+import java.util.UUID;
+
 public class Credentials {
     public final String email;
     public final String password;
+    @Nullable  public String username;
```

Comment:
Well incase of Log In we only need one of email or username. If we make this `NonNull` we'll have to provide an unused `username` incase of the `LoginFeatureTest`.